core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.html

```diff
@@ -1,40 +1,24 @@
-<p-tabView
-    [styleClass]="(featureFlagIsOn$ | async) && 'dot-template-builder__new-template-builder'">
+<p-tabView styleClass="dot-template-builder__new-template-builder">
     <p-tabPanel
         [header]="item.type === 'advanced' ? ('code' | dm) : ('design' | dm)"
         data-testId="builder">
-        <ng-container *ngIf="item.type === 'advanced'; else elseBlock">
+        @if (item.type === 'advanced') {
             <dot-template-advanced
                 (updateTemplate)="updateTemplate.emit($event)"
                 (save)="save.emit($event)"
                 (cancel)="cancel.emit()"
                 [didTemplateChanged]="didTemplateChanged"
                 [body]="item.body"></dot-template-advanced>
-        </ng-container>
-        <ng-template #elseBlock>
-            <ng-template #enabledComponent>
-                <dotcms-template-builder-lib
-                    (templateChange)="onTemplateItemChange($event)"
-                    [layout]="item.layout"
-                    [template]="{ themeId: item.theme }"
-                    [containerMap]="item.containers"
-                    data-testId="new-template-builder">
-                    <dot-global-message toolbar-left></dot-global-message>
-                </dotcms-template-builder-lib>
-            </ng-template>
-            <ng-template #disabledComponent>
-                <dot-edit-layout-designer
-                    (saveAndPublish)="saveAndPublish.emit($event)"
-                    (updateTemplate)="onTemplateItemChange($event)"
-                    (save)="save.emit($event)"
-                    [theme]="item.theme"
-                    [disablePublish]="item.live"
-                    [layout]="item.layout"></dot-edit-layout-designer>
-            </ng-template>
-            <ng-container
-                *dotShowHideFeature="featureFlag; alternate: disabledComponent"
-                [ngTemplateOutlet]="enabledComponent"></ng-container>
-        </ng-template>
+        } @else {
+            <dotcms-template-builder-lib
+                (templateChange)="onTemplateItemChange($event)"
```

Comment:
Here the output send a `DotTemplateDesigner` but the `onTemplateItemChange` receive a `DotTemplateItem` and does not match.

Reply:
This an architectural problem that I don't really wanna solve in this PR. I added some missing logic there tho